packages/inula/src/external/JSXElement.ts

```
@@ -213,8 +227,8 @@ export function cloneElement(element, setting, ...children) {
 }
 
 // 检测结构体是否为合法的Element
-export function isValidElement<P>(element: KVObject | null | undefined): element is InulaElement<P> {
-  return !!(element && element.vtype === TYPE_COMMON_ELEMENT);
+export function isValidElement<P>(element: {} | null | undefined): element is InulaElement<P> {
```

ESLint

[error] Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.
- If you really want a type meaning "any non-nullish value", you probably want `NonNullable<unknown>` instead.

(@typescript-eslint/ban-types)